types/http/status.pp

```diff
@@ -0,0 +1 @@
+type Stdlib::Http::Status = Integer[100, 599]
```

Comment:
# @summary A valid HTTP status code per RFC9110
# @see https://httpwg.org/specs/rfc9110.html#overview.of.status.codes
```suggestion
type Stdlib::Http::Status = Integer[100, 599]
```